src/CityDisplay.js

```diff
@@ -0,0 +1,89 @@
+import './App.css';
+import React from 'react';
+import axios from 'axios';
+
+class CityDisplay extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayCityData: false,
+      cityData: [],
+      userSearch: '',
+      errMsg: '',
+      errRender: false,
+
+    }
+  }
+
+  handleExplore = async () => {
+
+    try {
+      let cityResults = await axios.get('https://us1.locationiq.com/v1/search.php')
+
+      this.setState({
+        displayCityData: true,
+        cityData: cityResults.data.results
+      })
+    } catch (error) {
+      this.setState({
+        errRender: true,
+        errMsg: `An error has occurred: ${error.response.status}; ${error.response.data}`
+      })
+    }
+  }
+
+  handleCitySubmit = e => {
```

Comment:
Your handleCitySubmit would be serve better as a callback func for an onInput attribute in your input tag. Doing so would update your state per user keystroke. Remember that using onInput/onChange means you access the user data as event.target.value not event.target.name.value